backend/src/main/java/gov/cdc/usds/simplereport/service/email/EmailService.java

```diff
@@ -133,4 +133,23 @@ public String sendWithProviderTemplate(
 
     return emailProvider.send(mail);
   }
+
+  public String sendWithDynamicTemplate(
```

Comment:
Not a blocker, but might consider making the name of this method match the other one for sending with a dynamic template without variables (or maybe the other way around?... this name is probably more clear).

Reply:
sure thing, I'll swab the other method to this name